### PR TITLE
Apprenticeship copy edits

### PR DESCRIPTION
**Pull request by sofer**
Just some minor stylistic changes.

### CODE REVIEW

File: src/pages/documents/apprenticeships.md [hunk 1]
```diff
@@ -24,25 +24,25 @@ The [Founders and Coders curriculum](https://founders-and-coders.gitbook.io/cour
 We focus relentlessly on peer learning, pair programming and on creating an inclusive classroom environment that cultivates developers capable of working both respectfully and productively as part of a team.
 
 ### Community support
-We only ever run one cohort at any time, with a fixed cap of 16 learners, because we aren’t willing to trade-off scale for quality of learning. Our learners benefit from an extraordinary alumni community, dozens of whom volunteer every cohort by giving talks or offering mentorship and coaching. 
+We run cohorts of no more than 16 developers to ensure that the highest bar for learning is maintainted. Our learners benefit from an extraordinary alumni community, dozens of whom volunteer every cohort by giving talks or offering mentorship and coaching.
```
Comment (oliverjam) [inline src/pages/documents/apprenticeships.md:27]:
```suggestion
We run cohorts of no more than 16 developers to ensure that the highest bar for learning is maintained. Our learners benefit from an extraordinary alumni community, dozens of whom volunteer every cohort by giving talks or offering mentorship and coaching. 
```

Small typo. I quite like the new word "maintainted" though 😆

File: src/pages/documents/apprenticeships.md [hunk 2]
```diff
@@ -24,25 +24,25 @@ The [Founders and Coders curriculum](https://founders-and-coders.gitbook.io/cour
...
+It has always been our aim to promote inclusion, equity and diversity in tech and our graduates reflect that aim more with each passing year.
 
-## Why are we starting to do apprenticeships?
-Apprenticeships let us go further than our tuition-free programme for the benefit of students and employers.
+## Why apprenticeships?
+Providing apprenticeships is an exciting new development, allowing us to go further than our tuition-free programme for the benefit of students and employers alike.
 
 ### For students: making software development even more accessible
-By making it possible for students to earn an apprentice wage while they are learning, we will open up this career path to a much wider group of people for whom even a “free” course (with all the costs of living in London) has been previously unaffordable.
+Apprenticerships allow students to earn an apprentice wage while they are learning, opening up this career path to a much wider group of people for whom even a “free” course (with all the costs of living in London) has been previously unaffordable.
```
Comment (oliverjam) [inline src/pages/documents/apprenticeships.md:36]:
```suggestion
Apprenticeships allow students to earn an apprentice wage while they are learning, opening up this career path to a much wider group of people for whom even a “free” course (with all the costs of living in London) has been previously unaffordable.
```
Another typo

**Review by oliverjam — CHANGES_REQUESTED** [review]
Looks good—just a couple of typos

**Comment by sofer** [comment]
Thanks @oliverjam

**Pull request merged**